### ISSUE TITLE
You want to make a fair comparison of baselines, why don't you tune hyper-parameters?

### ISSUE THREAD

**Issue by chenchongthu** [xue-pai/Open-Match-Benchmark#6]
I noticed that you have reported the results of our ENMF on Amazon, Yelp, and Gowalla datasets, but the hyper-parameters "dropout"  and "negative_weight " seem not to be tuned?  

We have highlighted in our Github that the above two parameters are important for our results. If you do not tune the parameters and only use the default settings, then the results are meaningless.

Besides, I don't understand why in your experiment the training of ENMF is so slow on Amazon dataset. On our GTX 1080,  the training time on Amazon only takes about 30 seconds for each epoch.
